packages/lzip.rb

```diff
@@ -3,27 +3,19 @@
 class Lzip < Package
   description 'Lzip is a lossless data compressor with a user interface similar to the one of gzip or bzip2.'
   homepage 'https://www.nongnu.org/lzip/lzip.html'
-  version '1.22'
+  version '1.22-1'
   license 'GPL-2+'
   compatibility 'all'
   source_url 'https://download.savannah.gnu.org/releases/lzip/lzip-1.22.tar.gz'
   source_sha256 'c3342d42e67139c165b8b128d033b5c96893a13ac5f25933190315214e87a948'
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/lzip/1.22_armv7l/lzip-1.22-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/lzip/1.22_armv7l/lzip-1.22-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/lzip/1.22_i686/lzip-1.22-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/lzip/1.22_x86_64/lzip-1.22-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '34905a03071ba31b215c5aa5f769d2d835063cc3959df47dfdbd38654c1785a0',
-     armv7l: '34905a03071ba31b215c5aa5f769d2d835063cc3959df47dfdbd38654c1785a0',
-       i686: '9db33a3fd08f23985a9f9fc9c86b647886b394622b39f61db2cea62464091900',
-     x86_64: '77c7eb34fe9a72694f1c9fe10f8c27e3205526f90b7bd55c6e659dd34e7b60f8',
-  })
```

Comment:
We'll need new binaries before this can be merged.  I can do it.

Reply:
Thanks 🥺